impress-org/givewp
# PayPal commerce related elements matche Figma design

Resolves #5003

## Description
This PR adds styling fix to match Figma designs. I added a detailed description of styling bugs in the issue description.

## Affects
This pr does not affect exiting pr.

## Visuals
![image](https://user-images.githubusercontent.com/1784821/89438953-190b5d80-d767-11ea-9730-5bff11de5dad.png)
![VA5uQmCJZw](https://user-images.githubusercontent.com/1784821/89439034-3b9d7680-d767-11ea-8ad1-0ae71fbf4d26.gif)
![image](https://user-images.githubusercontent.com/1784821/89439167-68518e00-d767-11ea-8684-ba0cc4ceacd3.png)


## Pre-review Checklist

## Testing Instructions
Styling fixes added to the frontend for legacy and multi-step form template and admin setting screen, so you can visit both pages to check fixes.
